cfgov/unprocessed/css/organisms/snippet-list.less

```diff
@@ -35,11 +35,11 @@
     }
 
     &-item-title {
-        .webfont-medium();
+        .u-webfont-medium();
     }
 
     &-action-list {
-        .list__horizontal();
+        .m-list__horizontal();
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/3092/commits/f018ff24573f53a3759a187b393fd7dc3e56183c